cl/cltypes/beacon_blob_side_car.go

```diff
@@ -48,7 +49,7 @@ func (b *BlobSideCar) EncodeSSZ(buf []byte) ([]byte, error) {
 
 func (b *BlobSideCar) DecodeSSZ(buf []byte, version int) error {
 	if len(buf) < b.EncodingSizeSSZ() {
-		return ssz.ErrLowBufferSize
+		return fmt.Errorf("[BlobSideCar] err: %v", ssz.ErrLowBufferSize)
```

Comment:
need use %w - then errors.Is will work with wraped error. see https://go.dev/blog/go1.13-errors